Foundation/Process.swift

```diff
@@ -232,23 +232,238 @@ open class Process: NSObject {
         }
     }
 
+    #if os(Windows)
+    private func _socketpair() -> (first: SOCKET, second: SOCKET) {
```

Comment:
Does CFSocket not work on Windows as desired?

Reply:
There is no `socketpair` on Windows, this emulates the `socketpair` API.

Reply:
Ah!

Reply:
Good by me :)